source/cloud/azure/azure-ml-multi-node.md

```diff
@@ -0,0 +1,72 @@
+# Azure ML Compute cluster
+
+Launch Azure's [ML Compute cluster](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-create-attach-compute-cluster?tabs=python) to distribute your RAPIDS training jobs across single or multi-GPU compute nodes. The Compute cluster scales up automatically when a job is submitted, and executes in a containerized environment, packaging your model dependencies in a Docker container.
+
+## Pre-requisites
+
+All you need to get started is an Azure machine learning workspace and development environment must be installed with [Python SDK v2](https://learn.microsoft.com/en-us/python/api/overview/azure/ai-ml-readme?view=azure-python)or [Azure CLI v2](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-configure-cli?tabs=public). You could install Azure Machine Learning on your local computer but is recommended to use Azure's compute instance that is fully configured with all necessary packages and tools.
+
+### Instantiate workspace
+
+If using the Python SDK, connect to your workspace either by explicitly providing the workspace details or load from `config.json` file.
+Make sure to replace `subscription_id`,`resource_group`, and `workspace_name` with your own.
+
+```console
+# Get a handle to the workspace
+ml_client = MLClient(
```

Comment:
Where do I import `MLClient` from?

Reply:
Using the `rapids` environment in the compute instance I had to run

```python
!pip install azure-ai-ml
from azure.ai.ml import MLClient
from azure.identity import DefaultAzureCredential
```

Reply:
apologies, i will include the imports in the snippet...